src/Backend/QueueDispatcherInterface.php

```diff
@@ -26,7 +26,7 @@ interface QueueDispatcherInterface
     /**
      * Get a backend by message type.
      *
-     * @param string $type
+     * @param string|null $type
```

Comment:
This method is returning a backend for the corresponding type, so it seems weird to pass `null`.

Why this change ? Maybe the check should be before calling this method instead ?

Reply:
I did these changes based on its current use:

https://github.com/sonata-project/SonataNotificationBundle/blob/989b954e5d7fa9b546d21ce2fe92fefab6927a60/src/Command/CleanupCommand.php#L47

https://github.com/sonata-project/SonataNotificationBundle/blob/989b954e5d7fa9b546d21ce2fe92fefab6927a60/src/Backend/MessageManagerBackendDispatcher.php#L61-L67

Reply:
We could add a comment maybe ?
And it would be nice to change the signature to `public function getBackend($type = null)` but it's BC, so we can only add a NEXT_MAJOR comment.